cmd/tailscale/cli/update.go

```diff
@@ -333,6 +333,73 @@ func (up *updater) updateMacSys() error {
 	return errors.New("The 'update' command is not yet implemented on macOS.")
 }
 
+func (up *updater) updateMacAppStore() error {
+	fmt.Println(`This is the macOS App Store version of Tailscale. Updates are managed via the
+App Store, and the program 'softwareupdate'.
+
+See https://tailscale.com/s/unstable-clients to use TestFlight or to install
+the non-App Store version.`)
```

Comment:
nit: I wonder if this disclaimer is useful to the user, now that the command will deal with running the update. Maybe we can just omit it?